pages/finance/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import { Column, RestTable } from 'mobx-restful-table';
+import { InferGetServerSidePropsType } from 'next';
+import { compose, translator } from 'next-ssr-middleware';
+import { PureComponent } from 'react';
+import { Container } from 'react-bootstrap';
+
+import PageHead from '../../components/PageHead';
+import { Bill, BillModel } from '../../models/Bill';
+import { i18n } from '../../models/Translation';
+
+const billStore = new BillModel();
+
+export const getServerSideProps = compose<{ id: string }, { bill: Bill }>(
+  async ({ params }) => {
+    const bill = await billStore.getOne(params!.id);
+
+    return {
+      props: JSON.parse(JSON.stringify(bill)),
+    };
+  },
+
+  translator(i18n),
```

Comment:
```suggestion
export const getServerSideProps = compose<{}, { list: Bill[] }>(
  translator(i18n),
  async () => {
    const list = await new BillModel().getList();

    return {
      props: JSON.parse(JSON.stringify({ list })),
    };
  },
```
1. 后端请求基本都不能共享实例
2. 中间件的顺序是没有 `next()` 调用的放最后
3. 你做的是列表页，哪里有参数查单个？